website/source/docs/provisioners/chef.html.markdown

```diff
@@ -77,6 +77,12 @@ The following arguments are supported:
   subdirectory called `logfiles` created in your current directory. The filename will be
   the `node_name` of the new node.
 
+* `use_policyfile (boolean)` - (Optional) Use Chef Policyfiles.
+
+* `policy_group (string)` - (Optional) Chef  policy group.
+
+* `policy_name (string)` - (Optional) Chef policy name.
```

Comment:
Should have more detail on what should be included in this string, and that it is required if `use_policyfile` is set (ref: https://github.com/hashicorp/terraform/pull/3329/files#diff-033150dd6357bb8d6e1a2c44b80d689bR195)